src/System.Runtime/src/System/Runtime/CompilerServices/CompilationRelaxations.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices 
+{
+    [Flags]
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public enum CompilationRelaxations : int
```

Comment:
Runtime looks at CompilationRelaxationsAttribute, and CoreLib impl itself is marked with it. If you pull it out and stop marking it with CompilationRelaxationsAttribute, it will change behavior... .